functions/src/routes/express_open.ts

```diff
@@ -6,7 +6,7 @@ import app_open from "../express_configs/express_open";
 import { Request, Response } from "express";
 
 import * as challengeFunctions from "../challenge/challenge";
-import * as hacktoberfestFunctions from "../custom/hacktoberfest";
+import * as hacktoberfestFunctions from "../deprecated/hacktoberfest";
```

Comment:
Will deprecated functionality still be packaged with production releases or will this change soon?

Reply:
Good question, a majority of the code that was deployed for Hacktoberfest 2020 has already been removed from production releases. However, I've left this one included because those APIs are required for the content in the [HTF Development Workshop](https://github.com/acmutd/htf-development) to work. 

This update marks them deprecated for developers to recognize that those APIs are no longer being maintained & to help clean up the repo. Open to better alternatives/suggestions if there are any.